### PR TITLE
Fixed lint errors

### PR DESCRIPTION
**Pull request by afinkenstadt**
As reported here:  http://go-lint.appspot.com/github.com/golang/gddo/gddo-server

### CODE REVIEW

**Comment by afinkenstadt** [comment]
Done.

File: gddo-server/graph.go [hunk 1]
```diff
@@ -41,8 +41,7 @@ func renderGraph(pdoc *doc.Package, pkgs []database.Package, edges [][2]int) ([]
 	p := out.Bytes()
 	if i := bytes.Index(p, []byte("<svg")); i < 0 {
 		return nil, errors.New("<svg not found")
-	} else {
-		p = p[i:]
-	}
+    }
+    p = p[i:]
```
Comment (shurcooL) [inline gddo-server/graph.go]: This does not look `gofmt`ed. You should `gofmt` the code.

Comment (afinkenstadt) [inline gddo-server/graph.go]: Truly weird, as `set list` seemed to show the same line spacing as the lines surrounding it.

Fixed by direct edit on github.

**Comment by adg** [comment]
@afinkenstadt can you please run gofmt on these source files and update the pr?

**Comment by afinkenstadt** [comment]
Yes, once I re-create my go 1.4 environment.  Sorry for the delay.

**Comment by garyburd** [comment]
The templates must be updated to match this change.

**Comment by LaPingvino** [comment]
#239 solved, so this one can be closed

**Pull request closed without merging**